server.js

```diff
@@ -9,6 +9,7 @@ const run = async () => {
   const { port, enableHttpsForDev } = getConfig();
 
   const app = await build();
+  
   if (enableHttpsForDev) {
```

Comment:
I wonder if we even need to support `https` in this service. We don't want to encourage anyone running it without a proper front end SSL termination like with Nginx..

Reply:
I'd put this into sign-and-verify for PlugFest2 I think so that we could run the Digital Bazaar test suite that they required for inclusion in their JFF CHAPI playground.  Their tests only worked with https endpoints.  And I think I had some other case where I had a front end served from https, from which I wanted to make AJAX calls to my local sign-and-verify, but same-origin rules (or something similar) don't allow AJAX calls from an https-served page to an http-served server. I think you might be right though.  Maybe it would be better to just provide instructions on how to serve from https for development, but without making it so easy that it might be tempting to just use it for production?

Reply:
I usually manage that using traefik in a docker-compose.yml see https://github.com/digitalcredentials/cred-admin/blob/main/docker-compose.yml and https://github.com/digitalcredentials/cred-admin/tree/main/traefik for an example.